root/usr/bin/nm-configure

```diff
@@ -55,7 +57,11 @@ gateway=$gw
 dns=$dns;
 dhcp-send-hostname=$sendhost
 [ipv6]
-method=ignore
+method=manual
+address1=$ip
+gateway=$gw
```

Comment:
If I'm reading this right this way the same value is used for address and gateway for both ipv4 and ipv6. Not sure if network manager is able to gracefully deal with ipv4 address being given as ipv6 address and vice versa, depending on which address the user inputs.

Reply:
Right, I think the most reasonable solution is to introduce new command `primary-static6`.

Reply:
Sounds good to me

Reply:
How about this?